app/models/episode_import.rb

```diff
@@ -18,25 +18,37 @@ class EpisodeImport < ApplicationRecord
   scope :having_duplicate_guids, -> do
     unscope(where: :has_duplicate_guid).where(has_duplicate_guid: true)
   end
-  scope :complete, -> { where(status: PodcastImport::COMPLETE) }
+  scope :complete, -> { where(status: COMPLETE) }
+  scope :finished, -> { where(status: [COMPLETE, FAILED]) }
+  scope :in_progress, -> { where.not(status: [COMPLETE, FAILED, CREATED]) }
+  scope :failed, -> { where(status: FAILED) }
 
   before_validation :set_defaults, on: :create
 
   validates :entry, :guid, presence: true
 
+  enum :status, {
+    created: CREATED,
+    complete: COMPLETE,
+    failed: FAILED,
+    saved: SAVED,
+    retrying: RETRYING,
+    audio_saved: AUDIO_SAVED
+  }, prefix: true
+
   def unlock_podcast
     if podcast_import.finished?
       podcast_import.podcast.update(locked: false)
     end
   end
 
   def retry!
-    update(status: PodcastImport::RETRYING)
+    update(status: RETRYING)
```

Comment:
i'm not sure why these are not the bang version of `update`, so i tried to leave as is in case that was a specific choice.

Reply:
Yeah, that is odd.  I'd be fine with `status_retrying!` here.  It's surprising to have the non-`!` version happening inside a method with a `!` on it.

We'll catch more of these inconsistencies after some in-depth QA, once we hit feature parity! (I hope).